src/example/components/basics/CameraIcon.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {Icon, IconProps} from 'react-native-elements';
+
+type SpecificIconType = Omit<IconProps, 'name' | 'type'>;
+
+// React Native Elementsを使用したアイコンのサンプル
+const GearIcon = ({color = '#9D8CB8', ...props}: SpecificIconType) => {
```

Comment:
Filename is `CameraIcon.tsx`. Remove the icon name from variable name so that they can be easily reused.

```suggestion
const Icon = ({color = '#9D8CB8', ...props}: SpecificIconType) => {
```

Reply:
I fixed this issue in all icon components.